### PR TITLE
feat(cli): Token Support

### PR DESCRIPTION
**Pull request by xoscar**
This PR adds token support for the `tracetest start` command

## Changes

- Adds new env variable to support the TOKEN
- Adds config to replace jwt and token from the given flags

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Loom video
https://www.loom.com/share/a1405279a47a429cae4fe0bb8df25233

### CODE REVIEW

File: cli/cmd/start_cmd.go [hunk 1]
```diff
@@ -49,6 +52,7 @@ func init() {
 	startCmd.Flags().StringVarP(&saveParams.organizationID, "organization", "", "", "organization id")
 	startCmd.Flags().StringVarP(&saveParams.environmentID, "environment", "", "", "environment id")
 	startCmd.Flags().StringVarP(&saveParams.agentApiKey, "api-key", "", "", "agent api key")
+	startCmd.Flags().StringVarP(&saveParams.tokenApiKey, "token", "", defaultToken, "token api key")
```
Comment (jorgeepc) [inline cli/cmd/start_cmd.go:55]: `cli-api-key`instead of `token`?

Comment (mathnogueira) [reply cli/cmd/start_cmd.go]: I'm kind of late to this discussion. But I think that `cli-api-key` is too close to `api-key` and it's hard to understand what's the difference between them. 

I would suggest using either `token` or a more meaningful name for this flag (which I have no suggestions yet). The idea behind using `token` is that the user will get the value from the `Token` tab, so will be more intuitive for the user to create a connection between the value and the flag they need to use.

**Review by mathnogueira — APPROVED** [review]
Left a comment on the flag name discussion. It's not a blocker

File: cli/cmd/start_cmd.go (hunk 1 above)
Comment (xoscar) [reply cli/cmd/start_cmd.go]: I agree with this statement too, I think we can go with token as it matches the SaaS cloud, then in the future we can decide to change it

**Pull request merged**